alibi_detect/cd/pytorch/lsdd.py

```diff
@@ -76,12 +77,7 @@ def __init__(
         self.meta.update({'backend': 'pytorch'})
 
         # set backend
```

Comment:
@kuutsav this isn't something you've introduced, but since you're already making changes to these sections, would you be able to change the `# set backend` to `# set device` here please?